Assets/!/Scripts/PatrollingScript.cs

```diff
@@ -49,4 +51,4 @@ private IEnumerator Wait(float waitTime)
         yield return new WaitForSeconds(waitTime);
         _isWaiting = false;
     }
-}
+}
```

Comment:
Missing line break

Reply:
Why cant I resolve it from this panel like the other options?

Reply:
Resolved it now